obspy/io/nlloc/core.py

```diff
@@ -321,6 +321,18 @@ def _read_single_hypocenter(lines, coordinate_converter, original_picks,
     ou.azimuth_max_horizontal_uncertainty = hor_unc_azim
     ou.preferred_description = str("uncertainty ellipse")
```

Comment:
This could in principle below inside that if clause be set to `ou.preferred_description = "confidence ellipsoid"` but maybe it's easier to leave it with the more easy to digest ellipse?